spring-cloud-stream-reactive/dependency-reduced-pom.xml

```diff
@@ -0,0 +1,144 @@
+<?xml version="1.0" encoding="UTF-8"?>

```

Comment:
I noticed this file keeps getting generated every time we run the build. May be we better add here. Is that the idea? /cc @mbogoevici